snaps/openvino/consumer-intel-npu-driver-and-openvino-toolkit-2404/README.md

```diff
@@ -23,19 +23,25 @@ sudo snap install --beta intel-npu-driver
 sudo snap install --dangerous ~/openvino-toolkit-2404_2024.4.0_amd64.snap
 ```
 
-Connect interfaces:
+Connect the following interfaces manually:
 
 ```
-sudo snap connect openvino-python:intel-npu intel-npu-driver:intel-npu
 sudo snap connect openvino-python:npu-libs intel-npu-driver:npu-libs
 sudo snap connect openvino-python:openvino-libs openvino-toolkit-2404:openvino-libs
-sudo snap connect openvino-python:opengl
-sudo snap connect openvino-python:home
-sudo snap connect openvino-python:network
-sudo snap connect openvino-python:network-bind
 sudo snap connect openvino-python:hugepages-control
 ```
 
+The following interfaces should auto-connect:
```

Comment:
how that we do not to manually connect these anymore ?

Reply:
Some of them auto-connect (network, network-bind, opengl) per the snapd documentation: https://snapcraft.io/docs/supported-interfaces

The other two (home and custom-device) I just noticed autoconnect when I last tested on a freshly provisioned system, but I have doubts now because that is inconsistent with the docs. I will test again later and report back.

Reply:
@hector-cao I just checked again and TBH I'm not sure why home and custom-device are auto-connecting, but I verified that they are.